modules/git/command.go

```diff
@@ -130,7 +132,9 @@ func (c *Command) RunInDirTimeoutEnvFullPipelineFunc(env []string, timeout time.
 	}
 
 	// TODO: verify if this is still needed in golang 1.15
```

Comment:
```suggestion
```

Reply:
perhaps we should actually verify it first... :stuck_out_tongue: